src/v2/codegen/mod.rs

```diff
@@ -64,6 +64,8 @@ pub struct CrateMeta {
     pub authors: Option<Vec<String>>,
     /// Whether we're planning to emit a lib, app or module.
     pub mode: EmitMode,
+    /// Whether or not to make this a root crate.
+    pub root: bool,
```

Comment:
Can we just make this `no_root`? Because, this would otherwise be a breaking change for `EmitterState::default()`, as the default for `root` would be `false`.

Reply:
latest version has this change. Let me know if you'd like the commits squashed.

Reply:
Nah, it's alright, I'll squash it in merge :smile: